src/commands/hardis/org/user/freeze.ts

```diff
@@ -93,115 +92,108 @@ export default class OrgUnfreezeUser extends SfdxCommand {
+    if (userlistraw.length === 0) {
+      const outputString = `[sfdx-hardis] No matching user records found for all profile  except ${exceptFilter}`;
       uxLog(this,c.yellow(outputString));
       return { deleted: [], outputString };
     }
 
-    // Simplify results format & display them
-    const records = []/*recordsRaw.map((record: any) => {
+    let userlist = userlistraw.map((record: any) => {
       return {
-        Id: record.Id,
-        isFrozen: record.isfrozen
+        Name: record.Name,
+        Profile: record.Profile.Name
       };
-    });*/
+    });
     uxLog(this,
-      `[sfdx-hardis] Found ${c.bold(records.length)} records:\n${c.yellow(columnify(records))}`
+      `[sfdx-hardis] Found ${c.bold(userlist.length)} records:\n${c.yellow(columnify(userlist))}`
     );
 
-    // Perform update
-   // const udapted = [];
-    //const updateErrors = [];
+    userlistraw = [];
     if (
       !prompt ||
       (await this.ux.confirm(
         c.bold(
-          `[sfdx-hardis] Are you sure you want to update this list of records in ${c.green(
+          `[sfdx-hardis] Are you sure you want to freeze this list of records in ${c.green(
             this.org.getUsername()
           )} (y/n)?`
         )
       ))
     ) {
-     /* for (const record of records) {
-        const deleteCommand =
-          'sfdx force:data:record:delete' +
-          ' --sobjecttype Flow' +
-          ` --sobjectid ${record.Id}` +
-          ` --targetusername ${username}` +
-          ' --usetoolingapi';
-        const deleteRes = await execSfdxJson(deleteCommand, this, {
-          fail: false,
-          output: false,
-          debug: debugMode
-        });
-        if (!(deleteRes.status === 0)) {
-          this.ux.error(
-            c.red(
-              `[sfdx-hardis] Unable to perform deletion request: ${JSON.stringify(
-                deleteRes
-              )}`
-            )
-          );
-          deleteErrors.push(deleteRes);
-        }
-        deleted.push(record);
-      }*/
+
+        apexcode = 'list<userLogin> userLoginList = ['+queryUser+']; \n'+
+        'Set<Id> userIdList = new Set<Id>();\n'+
+        'if(userLoginList != null && userLoginList.size()> 0){\n'+
+            'for(UserLogin userfromList : userLoginList){\n'+
+                'userfromList.isFrozen = true;\n'+
+                'userIdList.add(userfromList.UserId);\n'+
+            '}\n'+
+        '}\n'+
+        'upsert userLoginList;\n'+
+        'list<User> userList = [SELECT Id,Name,Profile.Name FROM User WHERE Id IN :userIdList];\n'+
+        'system.debug(\'OUTPUTVALUE=\'+JSON.serialize(userList)+\'END_OUTPUTVALUE\'); \n';
+        await fs.readFileSync(path.join(__dirname,'./apex-freeze.apex'),'utf8');
+        const targetFile = path.join(__dirname,'apex-freeze.apex');
+        await fs.writeFile(targetFile,apexcode);
+      
+        const apexScriptCommand = `sfdx force:apex:execute -f "${targetFile}" --targetusername ${username}`;
```

Comment:
What about a method in deployUtils.ts ? :)
const result = await executeApex(pathToApexFile) :p